ako-gateway-api/nodes/avi_model_l7_translator.go

```diff
@@ -347,11 +339,13 @@ func buildPoolWithBackendExtensionRefs(key string, poolNode *nodes.AviPoolNode,
 					continue
 				}
 				healthMonitorRefsSet.Insert(uuid.(string))
-			} else if filter.ExtensionRef.Kind == akogatewayapilib.RouteBackendExtensionKind {
+			} else if filter.ExtensionRef.Kind == akogatewayapilib.RouteBackendExtensionKind && !firstRBEProcessed {
```

Comment:
It seems behaviour is different for 
1. First define RBE with HM and then HM CRD on Extension Ref.
2. First define HM CRD and then RBE with HM on Extension Ref.